hydra-access-controls/lib/hydra/user.rb

```diff
@@ -18,7 +18,7 @@ module ClassMethods
     # By default, uses the unique identifier specified in by devise authentication_keys (ie. find_by_id, or find_by_email).  
     # You must have that find method implemented on your user class, or must override find_by_user_key
```

Comment:
Need to update these comments that are no longer instructional.  They also may reveal the motivation of using the more particular method name (so as to allow overriding but not to mess with the core `find_by`).